build.gradle

```diff
@@ -140,3 +144,15 @@ javadoc {
     failOnError = false
     exclude "**/impl/**";
 }
+
+def getProjectVersion() {
+    String ver = connectorVersion
+    if (ver.contains("-SNAPSHOT")) {
+        String versionLabel = ver.substring(0, ver.indexOf("-SNAPSHOT"))
+        def count = grgit.log(includes:['HEAD']).size()
+        def commitId = "${grgit.head().abbreviatedId}"
+        int curTime = new Date().getTime()/1000
```

Comment:
It is convenient to have the timestamp to make the build version unique if we have a need to manually test the publishing step multiple times for the same commit id.

Reply:
I am a bit skeptical of including the timestamp, because a) our goal is to link to specific commits, not to 'dirty' builds and b) jcenter assumedly keeps numerous builds for the same snapshot version.